c_pal_ll/win32/devdoc/sync_win32_requirements.md

```diff
@@ -26,17 +29,43 @@ MOCKABLE_FUNCTION(, WAIT_ON_ADDRESS_RESULT, wait_on_address, volatile_atomic int
 
 **SRS_SYNC_WIN32_24_003: [** If `WaitOnAddress` succeeds, `wait_on_address` shall return `WAIT_ON_ADDRESS_OK`. **]**
 
+## wait_on_address_64
+
+```c
+MOCKABLE_FUNCTION(, WAIT_ON_ADDRESS_RESULT, wait_on_address_64, volatile_atomic int64_t*, address, int64_t, compare_value, uint32_t, timeout_ms)
+```
+**S_R_S_SYNC_WIN32_05_001: [** `wait_on_address_64` shall call `WaitOnAddress` from the Windows API **]**
```

Comment:
>I  [](http://example.com/codeflow?start=98&length=2)

Missing ending .
Also this is somewhat inconsistent with what we spec for 32 bit in SRS_SYNC_WIN32_24_001. Consider normalizing so they look similar.

Reply:
Fixed.